### PR TITLE
IMPB-1499: Fixed the email symbol sanitation and submission actions.

### PR DESCRIPTION
**Pull request by KashIDX**
The issue was related to the lead email being broken on submission, in addition to that there's a QOL change to how the page reacts after the lead is successfully submitted.

### CODE REVIEW

**Review by rdowner22 — COMMENTED** [review]
`IMPB-1452:` Should we also remove the social pro advertisements from both `readme` files?

File: add-ons/listings/includes/listing-templates/listing-map-location.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+/**
+ * Loads the listing location on the map.
+ * 
+ * @return mixed
+ */
+function load_listing_on_map ( $post, $options ) {
+    $map_info_content = sprintf( '<p style="font-size: 14px; margin-bottom: 0;">%s<br />%s %s, %s</p>', get_post_meta( $post->ID, '_listing_address', true ), get_post_meta( $post->ID, '_listing_city', true ), get_post_meta( $post->ID, '_listing_state', true ), get_post_meta( $post->ID, '_listing_zip', true ) );
+
+    ( $options['wp_listings_gmaps_api_key'] ) ? $map_key = $options['wp_listings_gmaps_api_key'] : $map_key = '';
+    wp_enqueue_script( 'idxb-google-maps', 'https://maps.googleapis.com/maps/api/js?key=' . $map_key . '&loading=async&callback=initialize&libraries=marker', [], '1.0', 'async' );
```
Comment (rdowner22) [inline add-ons/listings/includes/listing-templates/listing-map-location.php:12]: I am not sure if we should be enqueuing the script at all if there is no key to use.

File: assets/js/idx-leads.js [hunk 2]
```diff
@@ -199,7 +199,8 @@ jQuery(document).ready(function($) {
 			},
 			success: function( result ) {
 				if( $.isNumeric( result ) ) {
-					window.location.href = leadurl + result;
+					alert('Lead Successfully Added');
+					window.location.reload(true);
```
Comment (rdowner22) [inline assets/js/idx-leads.js:203]: I assume this `alert` was just for testing. I noticed when working with the omnibar, just setting `window.location = 'newurl'` reloaded the page. Not saying it needs to be removed, but I am curious if the reload true makes a difference in the functionality.

**Review by rdowner22 — CHANGES_REQUESTED** [review]

File: assets/js/idx-leads.js (hunk 2 above)
Comment (KashIDX) [reply assets/js/idx-leads.js:203]: I left the alert in there as some kind of visual queue that the lead was added.

File: add-ons/listings/includes/listing-templates/listing-map-location.php (hunk 1 above)
Comment (KashIDX) [reply add-ons/listings/includes/listing-templates/listing-map-location.php:12]: Good point. I'll add a check

File: assets/js/idx-leads.js (hunk 2 above)
Comment (KashIDX) [reply assets/js/idx-leads.js:203]: Without the alert the page just reloads and empties the field

**Comment by KashIDX** [comment]
> `IMPB-1452:` Should we also remove the social pro advertisements from both `readme` files?

I address this in a new commit.

File: assets/js/idx-leads.js (hunk 2 above)
Comment (KashIDX) [reply assets/js/idx-leads.js:202]: I removed the alert after seeing that there isn't an alert for other actions.

**Review by rdowner22 — DISMISSED** [review]

**Review by rdowner22 — APPROVED** [review]

**Pull request merged**